src/components/css/booklist.css

```diff
@@ -0,0 +1,176 @@
+* {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.container-all {
+  width: 90%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+}
+
+.container-item {
+  display: flex;
+  flex-direction: row;
+  background-color: white;
+  border: 1px solid #e7e8e8;
+  gap: 14px;
+  padding: 2rem 1.188rem 1.625rem 1.688rem;
+}
+
+.container-item:nth-child(1) {
+  margin-top: 2rem;
+}
+
+.first-item {
+  flex: 2;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.second-item {
+  flex: 1;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 20px;
+  border-right: 1px solid #e8e8e8;
+}
+
+.thrid-item {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 12px;
+  margin-left: 25px;
+}
+
+.books-detail-1 {
+  display: flex;
+  flex-direction: column;
+  gap: 3px;
+  text-align: left;
+}
+
+.category {
+  font-size: 16px;
+  color: #666;
+  font-weight: bold;
+}
+
+.book-title {
+  margin: 0;
+}
+
+.book-author {
+  color: #0291ff;
+  font-size: 15px;
+}
+
+.books-detail-2 {
+  display: flex;
+  flex-direction: row;
+  gap: 2px;
+  text-align: left;
+}
+
+.books-detail-2 button {
+  background-color: white;
+  border: none;
+  color: #57b5fd;
+}
```

Comment:
- [OPTIONAL] I think it would be better to remove the padding from the left of the `comments` button to align the buttons and getting a design close to the Zeplin design guidelines 📌✨ Please fix that as well

![image](https://github.com/AliShahSafdari/bookstore/assets/106119105/340a6155-09df-4516-af2f-31bdc9bf10c0)